### DISCUSSION TITLE
Accessing custom fields inside notifications

### DISCUSSION THREAD

**Discussion by menop94** [swellstores/community#106] [Help] [answered]
I'm trying to customise our "Order confirmation" email that is sent out to include some custom fields that we have on the Order model. However, if I'm trying to use them like the other fields - {{ custom_field_name }} - they show up blank. Is there a way to achieve this?

**Comment by ericingram** [comment, marked as answer]
That should work as-is. Make sure the order you're testing with actually has the field defined. If you're sending a test email for example, it won't have a value for a custom field by default, but you can add a default value to the code like this:

```
{{ custom_field_name | default: 'Custom value' }}
```

**Reply by menop94** [reply]
Thank you for your answer!
However, it doesn't seem to work as-is. If I check the order in swell, under Order Activity there's a line saying "Sent order receipt to customer_email ..." and a button that says "View message". If I click on View message, I do see all the data properly. However, the email the customer is getting doesn't have any data for the custom fields. Any ideas why this might happen?

**Reply by ericingram** [reply]
It certainly should work. Share the details with our team at support@swell.is and we'll investigate. If there's a bug we'll fix it!
